tests/test_plotting.py

```diff
@@ -7,27 +7,20 @@
 def test_plot_logit_lens():
     pythia_125M_model = AutoModelForCausalLM.from_pretrained("EleutherAI/pythia-125m")
     tokenizer = AutoTokenizer.from_pretrained("EleutherAI/pythia-125m")
-    pythia_125M_lens = TunedLens(pythia_125M_model)
+    pythia_125M_lens = LogitLens.init_from_model(pythia_125M_model)
```

Comment:
Would prefer to make this just `from_model` since HuggingFace classes are always `.from_<whatever>`

Reply:
Okay agreed will change this.